android/app/src/main/java/org/fossasia/openevent/viewmodels/EditProfileActivityViewModel.java

```diff
@@ -0,0 +1,125 @@
+package org.fossasia.openevent.viewmodels;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+
+import org.fossasia.openevent.api.APIClient;
+import org.fossasia.openevent.data.auth.UploadImage;
+import org.fossasia.openevent.data.auth.User;
+import org.fossasia.openevent.dbutils.RealmDataRepository;
+import org.fossasia.openevent.utils.JWTUtils;
+import org.fossasia.openevent.utils.Utils;
+import org.json.JSONException;
+
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.schedulers.Schedulers;
+import timber.log.Timber;
+
+public class EditProfileActivityViewModel extends ViewModel {
+
+    public static final int IMAGE_UPLOAD_SUCCESSFUL =1;
+    public static final int IMAGE_UPLOAD_FAILED =2;
+    public static final int IMAGE_UPLOAD_COMPLETE =3;
+    public static final int USER_UPDATE_SUCCESSFUL=1;
+    public static final int USER_UPDATE_FAILED=2;
+    public static final int USER_UPDATE_COMPLETE=3;
```

Comment:
No need to define them for each method. Just rename them to SUCCESSFUL, FAILED, COMPLETE and reuse them in both the calls.

Reply:
you are saying so because updateUser() and uploadImage() will never be called together?